src/Storage/Controllers/CleanupController.cs

```diff
@@ -0,0 +1,246 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Interface.Models;
+using Altinn.Platform.Storage.Repository;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+
+namespace Altinn.Platform.Storage.Controllers
+{
+    /// <summary>
+    /// Handles cleanup of storage data
+    /// </summary>
+    [Route("storage/api/v1/cleanup")]
+    [ApiController]
+    public class CleanupController : ControllerBase
```

Comment:
We don't want these actions to show up in the OpenAPI spedification. 
```suggestion
    [ApiExplorerSettings(IgnoreApi = true)]
    public class CleanupController : ControllerBase
```